packages/bits/demo/src/components/demo/wizard-v2/wizard-dialog/wizard-dialog.example.component.html

```diff
@@ -1,96 +1,124 @@
 <ng-template #dialogWizard>
-    <div nui-busy [busy]="busy">
-        <div class="busy-content">
-            <nui-dialog-header i18n-title title="Wizard dialog" (closed)="closeDialog()"></nui-dialog-header>
-            <div class="dialog-body compact-mode">
-                <nui-wizard-horizontal>
+    <div nuiResizeObserver #dialog>
+        <div nui-busy [busy]="busy">
+            <div class="busy-content">
+                <nui-dialog-header i18n-title title="Wizard dialog" (closed)="closeDialog()"></nui-dialog-header>
+                <div class="dialog-body compact-mode">
+                    <nui-wizard-horizontal #wizard>
 
-                    <!--  content of the 1st step -->
-                    <nui-wizard-step-v2>
-                        <!-- define the label which is displayed on the header stepper -->
-                        <ng-template wizardStepLabel>Completed</ng-template>
+                        <!--  content of the 1st step -->
+                        <nui-wizard-step-v2>
+                            <!-- define the label which is displayed on the header stepper -->
+                            <ng-template wizardStepLabel>Completed</ng-template>
 
-                        <!-- content of the current step -->
-                        <span i18n>This is the first step of our horizontal wizard</span>.
+                            <!-- content of the current step -->
+                            <div class="step-content-wrapper">
+                                <span i18n>This is the first step of our horizontal wizard</span>.
+                            </div>
+                            <!-- content of the footer -->
+                            <ng-template wizardStepFooter>
+                                <button type="button" nui-button (click)="toggleBusy()">
+                                    Toggle busy
+                                </button>
+                                <ng-container *ngTemplateOutlet="wizardCancel"></ng-container>
+                                <ng-container *ngTemplateOutlet="wizardNext"></ng-container>
+                            </ng-template>
+                        </nui-wizard-step-v2>
 
-                        <!-- content of the footer -->
-                        <ng-template wizardStepFooter>
-                            <button type="button" nui-button (click)="toggleBusy()">
-                                Toggle busy
-                            </button>
-                            <ng-container *ngTemplateOutlet="wizardCancel"></ng-container>
-                            <ng-container *ngTemplateOutlet="wizardNext"></ng-container>
-                        </ng-template>
-                    </nui-wizard-step-v2>
+                        <!--  content of the 2nd step -->
```

Comment:
We may skip duplicating the same information for the rest of the steps since it was already described above and its all the same across the steps